generator/src/main/java/com/scottlogic/deg/generator/restrictions/NumericRestrictions.java

```diff
@@ -36,6 +36,23 @@ public boolean match(Object o) {
         return true;
     }
 
+    public boolean numericValuesAreInteger() {
+        if (min == null || max == null) {
+            return true;
```

Comment:
Logically if something doesn't exist you cannot say it is something?
Should we return `true` if we know nothing about `min` or `max`?

Reply:
This comes back to what we discussed about moving the null check into the NumericRestrictions class because the call is unsafe. You are correct that we can't make any assumptions but this was the existing logic